lib/odooauth.py

```diff
@@ -39,7 +41,12 @@ async def check_login(self, username, password):
             return False, False
         cookie=resp.cookies['session_id']
         result=resp.json()
-        if not result['result'].get('uid'):
+        if not 'result' in result:
```

Comment:
@tijnvandekam I was getting an error with the bouncer sometimes, this fixed it. Not sure if it's the same error you were getting.